### ISSUE TITLE
Table doesn't render Decimal values

### ISSUE THREAD

**Issue by stefan-sherwood** [zauberzeug/nicegui#901]
### Description

```py
from nicegui import ui
from decimal import Decimal

columns= [{ 'field': 'number', 'label': 'number' }]
rows=[{'number':1},{'number':float(1.1)},{'number':Decimal('1.1')}]

ui.table( columns=columns, rows=rows )
ui.run()
```

![image](https://user-images.githubusercontent.com/2094779/236505190-747cef80-189f-48b1-9711-c870a85684df.png)

Row with `Decimal` data type is blank. Other rows show fine.

I am using nicegui version 1.2.12 and I am pulling data from a MySQL database. Naturally, some of the data in my rows have the standard `Decimal` data type.

Note that `str( Decimal( '1.1') )` is `'1.1'`

I tried converting all of the Decimal data items to string but that breaks sorting and it feels like chasing my tail to implement my own sorting algorithm to work around this issue.

If this is intended behavior, I'd appreciate it if someone would advise me on how to get this data to display properly.

**Comment by stefan-sherwood** [comment]
Fastest turnaround time ever! Thanks!

**Comment by rodja** [comment]
We just released 1.2.13 with this bugfix 🏁
